src/main/java/ca/nrc/cadc/beacon/web/resources/NodeServerResource.java

```diff
@@ -0,0 +1,151 @@
+*                                       par écrit.
+*
+*  This file is part of the             Ce fichier fait partie du projet
+*  OpenCADC project.                    OpenCADC.
+*
+*  OpenCADC is free software:           OpenCADC est un logiciel libre ;
+*  you can redistribute it and/or       vous pouvez le redistribuer ou le
+*  modify it under the terms of         modifier suivant les termes de
+*  the GNU Affero General Public        la “GNU Affero General Public
+*  License as published by the          License” telle que publiée
+*  Free Software Foundation,            par la Free Software Foundation
+*  either version 3 of the              : soit la version 3 de cette
+*  License, or (at your option)         licence, soit (à votre gré)
+*  any later version.                   toute version ultérieure.
+*
+*  OpenCADC is distributed in the       OpenCADC est distribué
+*  hope that it will be useful,         dans l’espoir qu’il vous
+*  but WITHOUT ANY WARRANTY;            sera utile, mais SANS AUCUNE
+*  without even the implied             GARANTIE : sans même la garantie
+*  warranty of MERCHANTABILITY          implicite de COMMERCIALISABILITÉ
+*  or FITNESS FOR A PARTICULAR          ni d’ADÉQUATION À UN OBJECTIF
+*  PURPOSE.  See the GNU Affero         PARTICULIER. Consultez la Licence
+*  General Public License for           Générale Publique GNU Affero
+*  more details.                        pour plus de détails.
+*
+*  You should have received             Vous devriez avoir reçu une
+*  a copy of the GNU Affero             copie de la Licence Générale
+*  General Public License along         Publique GNU Affero avec
+*  with OpenCADC.  If not, see          OpenCADC ; si ce n’est
+*  <http://www.gnu.org/licenses/>.      pas le cas, consultez :
+*                                       <http://www.gnu.org/licenses/>.
+*
+*
+************************************************************************
+*/
+
+package ca.nrc.cadc.beacon.web.resources;
+
+
+import ca.nrc.cadc.beacon.web.restlet.JSONRepresentation;
+import ca.nrc.cadc.beacon.web.restlet.VOSpaceApplication;
+import ca.nrc.cadc.vos.Node;
+import ca.nrc.cadc.vos.NodeNotFoundException;
+import ca.nrc.cadc.vos.VOS;
+import ca.nrc.cadc.vos.VOSURI;
+import ca.nrc.cadc.vos.client.VOSpaceClient;
+import org.json.JSONException;
+import org.json.JSONWriter;
+import org.restlet.data.Status;
+import org.restlet.resource.Get;
+import org.restlet.resource.ResourceException;
+//import sun.jvm.hotspot.debugger.cdbg.AccessControl;
+
+import java.io.FileNotFoundException;
+import java.security.AccessControlException;
+
+public class NodeServerResource extends StorageItemServerResource
+{
+
+    @Get("json")
+    public void checkItemAccess() throws ResourceException
+    {
+//        try {
+
+            Node requestedNode = getNode(getCurrentItemURI(), VOS.Detail.max, 0);
+
+            // if it gets to here without throwing an error, check the writable property
```

Comment:
Format this code to remove commented out items.  Also move the comments that relate to what this class does to the class JavaDoc.